addons/sale_stock/models/sale_order.py

```diff
@@ -428,6 +428,10 @@ def write(self, values):
         if 'product_uom_qty' in values:
             lines = self.filtered(lambda r: r.state == 'sale' and not r.is_expense)
 
+        if 'product_packaging_id' in values:
+            for move_id in self.mapped('move_ids'):
+                move_id.product_packaging_id = values['product_packaging_id']
```

Comment:
I think the whole chain should be updated, so with something like
```suggestion
            self.move_ids.group_id.stock_move_lines.product_packaging_id = values['product_packaging_id']
```
Otherwise, it would not work with a multi-steps configuration
But I wonder what it should do if there is already some done quantities. imho, if at least one picking is already done, we could prevent the user from updating the packaging @odoo/rd-logistics (if we really want to fix the use case)